spec/requests/api/connect/v3/subscriptions/systems_controller_spec.rb

```diff
@@ -54,5 +54,54 @@
         it { is_expected.to match_array(%w[testhost0 testhost1 testhost2]) }
       end
     end
+
+    context 'with hwinfo parameters' do
```

Comment:
Please stick with a common `hw_info` name. The use of `hwinfo` and `hw_info` can get confusing.

Reply:
ok but I still have to use `params.permit(:hwinfo)` or `params.require(:hwinfo)` because that's how the param is named..